_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit, machines, ostree
+slug: cockpit-248
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248, Cockpit-Machines 247, and Cockpit-OSTree 187:
+
+
+## Metrics: Install missing packages
+
+[Cockpit 247](https://cockpit-project.org/blog/cockpit-247.html) introduced setting up a machine for Grafana metrics. However, this only worked if the `pcp` and `redis` packages were already installed. With this Cockpit version, the options will always work, and missing packages get installed as needed.
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages.png)
+
+![screenshot of install missing packages]({{ site.baseurl }}/images/248-metrics-install-missing-packages-2.png)
+
+This completes the story of [setting up PCP and Grafana metrics with Cockpit](https://cockpit-project.org/blog/pcp-grafana.html), have a look at that blog post for all the details.
```

Comment:
evil comma splice

Reply:
I updated this in https://github.com/cockpit-project/cockpit/pull/16009 in case you want to re-import.